### DISCUSSION TITLE
Behavior of mouse inputs not consistent with stable

### DISCUSSION THREAD

**Discussion by l4ssc** [ppy/osu#21099] [Q&A] [answered]
In osu! stable, clicking K1/K2 lifts M1/M2 while M1/M2 are held (but _not_ vice versa, where M1/M2 inputs are blocked when K1/K2 inputs are held). osu!lazer is inconsistent with this due to what I assume is the conflation of M1 and K1 into just "B1", where M1 will remain held when K1 is clicked and vice versa.

**Comment by peppy** [comment, marked as answer]
I think you might be misunderstanding. Can you show a video of what you think you're m seeing in stable?

**Reply by l4ssc** [reply]
[this is what I'm talking about. perhaps the input overlay is wrong here!](https://streamable.com/c605xk)

EDIT: yea, the actual behavior is the same for both; just the input overlay that's weird 🤔
